cobbler/items/image.py

```diff
@@ -1,5 +1,103 @@
 """
 Cobbler module that contains the code for a Cobbler image object.
+
+Changelog:
+
+Current Schema:
+    * arch: enums.Archs
+    * autoinstall: enums.VALUE_INHERITED
+    * breed: str
+    * file: str
+    * image_type: enums.ImageTypes
+    * network_count: int
+    * os_version: str
+    * boot_loaders: Union[list, str]
+    * menu: str
+    * display_name: str
+    * virt_auto_boot: bool
+    * virt_bridge: str
+    * virt_cpus: int
+    * virt_disk_driver: enums.VirtDiskDrivers
+    * virt_file_size: enums.VALUE_INHERITED
+    * virt_path: str
+    * virt_ram: int
+    * virt_type: enums.VirtType
```

Comment:
Except `virt_cpus` and `virt_path` all of the types have changed, either to enums or Union Types